src/main/java/org/folio/innreach/batch/contribution/service/ContributionJobRunner.java

```diff
@@ -179,7 +181,7 @@ public void runInstanceContribution(UUID centralServerId, Instance instance) {
     }
 
     runOngoing(centralServerId, (ctx, statistics) -> {
-      log.info("Ongoing: starting ongoing instance contribution job {}", ctx);
+      log.info("Ongoing: starting ongoing instance contribution job {}, centralServerId: {}", ctx, centralServerId);
 
       if (eligibleInstance) {
         log.info("Ongoing: contributing instance");
```

Comment:
All of the rest of the log statements in this method should have the instance id logged.

Reply:
Added Instance and item ids in every log statement.

Reply:
I think you misunderstood the comment. I'm suggesting that you log the instance ids in this method. A lot of the other logging that you have added is good though and shouldn't be removed unless I have suggestions about it. But here we care about whether these conditions have been hit in this method, for the given instance. This is key to debugging the P1 which is about ongoing contributions.